Spec/RestClient.swift

```diff
@@ -1744,6 +1744,24 @@ class RestClient: QuickSpec {
                     expect(fallbackRequests).toNot(beEmpty())
                     expect(fallbackRequests).to(allPass { extractURLQueryValue($0?.url, key: "request_id") == requestId })
                 }
+                
+                it("ErroInfo should has `requestId` property") {
```

Comment:
Typo here :). `ErrorInfo` not `ErroInfo`, and the grammar is slightly wrong, `should have` instead of `should has`.

Reply:
changed here 0edd476c52430ebfa9ceb6ea873555a8bad7486f